.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    paths:
+      - "**.rs"
+      - "Cargo.toml"
+      - "Cargo.lock"
+      - ".github/workflows/**"
```

Comment:
Can this be set to target pushes to `master` (or the default branch) as well, and can you please set paths to "*" for ease?

Reply:
Done. The double `*` is needed in the first rule though as a single wildcard doesn't match the `/` character. So it would only match all files inside the root folder that end in `*.rs`. `**.rs` would match any rust file inside any folder (`src/*.rs` as well as a `build.rs` script).